app/mod_api/models.py

```diff
@@ -213,20 +271,28 @@ def get_userinfo_by_id(user_id):
 
 
 class PublisherUser(db.Model):
+    """
+    This class is association object between user and publisher
+    as they have many to many relationship
+    """
     __tablename__ = 'publisher_user'
 
     id = db.Column(db.Integer, primary_key=True, autoincrement=True)
     created_at = db.Column(db.DateTime, default=datetime.datetime.utcnow)
     user_id = db.Column(db.Integer, ForeignKey('user.id'), primary_key=True)
     publisher_id = db.Column(db.Integer, ForeignKey('publisher.id'), primary_key=True)
 
-    role = db.Column(db.TEXT, nullable=False)  # role can be OWNER and MEMBER
+    role = db.Column(db.TEXT, nullable=False)
+    """role can only OWNER or MEMBER"""
```

Comment:
@subhankarb Oh! I was wron that it really restricts the input. And actually since v. 1.1 the Sqlalchemy standard "Enum" field does it! http://docs.sqlalchemy.org/en/latest/core/type_basics.html#sqlalchemy.types.Enum
 To disable native postgres restriction you can pass it `native_enum=False`